src/react-apps/applications/service-development/src/sharedResources/appCluster/getDeployments/getDeploymentsInAppClusterSagas.ts

```diff
@@ -1,36 +1,46 @@
 import { SagaIterator } from 'redux-saga';
-import { call, fork, takeLatest } from 'redux-saga/effects';
+import { delay } from 'redux-saga';
+import { call, fork, race, select, take } from 'redux-saga/effects';
 import { get } from '../../../../../shared/src/utils/networking';
 import { urls } from '../../../config/sharedConfig';
 import * as AppClusterActionTypes from '../appClusterActionTypes';
 import AppClusterDispatcher from '../appClusterDispatcher';
-import * as GetDeploymentsActions from './getDeploymentsInAppClusterActions';
 
-// GET DEPLOYMENT
-export function* getDeploymentsSaga({
-  env,
-  org,
-  repo,
-}: GetDeploymentsActions.IGetDeployments): SagaIterator {
-  try {
-    const result = yield call(get,
-      // tslint:disable-next-line:max-line-length
-      `https://${org}.apps.${env}.${urls.hostname.apps.test}/kuberneteswrapper/api/v1/deployments?labelSelector=release=${org}-${repo}`);
-    yield call(AppClusterDispatcher.getDeploymentsFulfilled, result, env);
-  } catch (err) {
-    yield call(AppClusterDispatcher.getDeploymentsRejected, err, env);
+const SelectEnvironments = (store: IServiceDevelopmentState) => store.configuration.environments.result;
+
+function* getDeploymentsIntervalSaga(): SagaIterator {
+  while (true) {
+    const { org, app } = window as Window as IAltinnWindow;
+    const environments: any = yield select(SelectEnvironments); // TODO: any
```

Comment:
Note this as something to do in the future

Reply:
Done